From d9cdb2119e869f91a7e42864a84342e8bf16131c Mon Sep 17 00:00:00 2001
From: Wolfgang Grandegger <wg@aries-embedded.de>
Date: Wed, 3 Aug 2022 16:59:28 +0200
Subject: [PATCH 11/13] ma5d4evk: fix names of the elements in the fitImage

The Kernel is now names 'kernel-1' and the FTD file 'fdt-<dtb-name>'.

Signed-off-by: Wolfgang Grandegger <wg@arie-embedded.de>
---
 include/configs/ma5d4evk.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/configs/ma5d4evk.h b/include/configs/ma5d4evk.h
index 4eca95b56c..b65420e49a 100644
--- a/include/configs/ma5d4evk.h
+++ b/include/configs/ma5d4evk.h
@@ -179,16 +179,16 @@
 		"fi\0"							\
 	"mmc_mmc="							\
 		"run fdtimg mmcload mmcargs addargs ; "			\
-		"bootm ${kernel_addr_r}:kernel@1 - ${kernel_addr_r}:fdt@${kernel_fdt}\0" \
+		"bootm ${kernel_addr_r}:kernel-1 - ${kernel_addr_r}:fdt-${kernel_fdt}\0" \
 	"mmc_nfs="							\
 		"run fdtimg mmcload nfsargs addip addargs ; "			\
-		"bootm ${kernel_addr_r}:kernel@1 - ${kernel_addr_r}:fdt@${kernel_fdt}\0" \
+		"bootm ${kernel_addr_r}:kernel-1 - ${kernel_addr_r}:fdt-${kernel_fdt}\0" \
 	"net_mmc="							\
 		"run fdtimg netload mmcargs addargs ; "			\
-		"bootm ${kernel_addr_r}:kernel@1 - ${kernel_addr_r}:fdt@${kernel_fdt}\0" \
+		"bootm ${kernel_addr_r}:kernel-1 - ${kernel_addr_r}:fdt-${kernel_fdt}\0" \
 	"net_nfs="							\
 		"run fdtimg netload nfsargs addip addargs ; "			\
-		"bootm ${kernel_addr_r}:kernel@1 - ${kernel_addr_r}:fdt@${kernel_fdt}\0" \
+		"bootm ${kernel_addr_r}:kernel-1 - ${kernel_addr_r}:fdt-${kernel_fdt}\0" \
 	"try_bootscript="						\
 		"mmc rescan;"						\
 		"if test -e mmc 1:1 ${bootscript} ; then "		\
-- 
2.34.1

